pkg/cmd/server/origin/master_config.go

```diff
@@ -221,7 +221,7 @@ func BuildMasterConfig(options configapi.MasterConfig) (*MasterConfig, error) {
 
 	kubeletClientConfig := configapi.GetKubeletClientConfig(options)
 
-	quotaRegistry := quota.NewAllResourceQuotaRegistry(privilegedLoopbackOpenShiftClient, privilegedLoopbackKubeClientset, kubeInformerFactory)
+	quotaRegistry := quota.NewAllResourceQuotaRegistry(informerFactory, privilegedLoopbackOpenShiftClient, privilegedLoopbackKubeClientset)
```

Comment:
This one already accepted informers, but just k8s ones, but I needed both k8s and origin ones. I've also moved this parameter from the end of this function call as the first argument, to make it similar to other methods. And since `quota.NewAllResourceQuotaRegistry` under covers creates evaluators which are now switched to informers, it's needed this way.